src/ui/ResultActions/ResultActionsMenu.ts

```diff
@@ -0,0 +1,126 @@
+import { Component } from '../Base/Component';
+import { ComponentOptions } from '../Base/ComponentOptions';
+import { IResultsComponentBindings } from '../Base/ResultsComponentBindings';
+import { $$ } from '../../utils/Dom';
+import { IQueryResult } from '../../rest/QueryResult';
+import { Assert } from '../../misc/Assert';
+import { Initialization } from '../Base/Initialization';
+import * as _ from 'underscore';
```

Comment:
I prefer if we `import {x,y,z} from underscore` instead of `import *`.

This will allow us to eventually do tree shaking.